### PR TITLE
Feature/add-item-to-pr-template

### LINKED ISSUE #1906: Lerna updates major versions of all our components
Found a possible problem:

Found at `file-upload`. It has a dependency to `input-file`. input-file had a breaking change (text of button is now a attribute, not a child). Lerna updated this breaking-change version of input-file (4.0.2). 

consequence: The red button of file-upload is not showing the text "DATEI HOCHLADEN" anymore. --> this bug is in Prod right now (and fixed by PR #1904 )!

Do we want to change lerna so that major versions are not updated automatically?

### PR DESCRIPTION
**Pull request by 01231**
Fixes #1906

# Merge Checklist:
- [ ] Code selfreview has been performed ([Best Practices](https://github.com/axa-ch/patterns-library/blob/develop/CONTRIBUTION.md#best-practices)).
- [ ] Tests are sufficient.
- [ ] Modifications still work in IE.
- [ ] Documentation is up to date.
- [ ] Changelog is up to date.
- [ ] Typescript typings for properties are up to date.
- [ ] Designers approved changes or no approval needed.

### CODE REVIEW

**Comment by 01231** [comment]
can I close this one #1906?

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -8,3 +8,4 @@ Fixes #
 - [ ] Changelog is up to date.
 - [ ] Typescript typings for properties are up to date.
 - [ ] Designers approved changes or no approval needed.
+- [ ] Check if peer dependencies of components still work.
```
Comment (danyball) [inline PULL_REQUEST_TEMPLATE.md]: I would suggest "Check if peer dependencies of components has any breaking changes"

Comment (01231) [reply PULL_REQUEST_TEMPLATE.md]: But this sentence could be true and the component would't have to work...

**Review by markus-walther — APPROVED** [review]
LGTM

**Review by danyball — APPROVED** [review]

**Pull request merged**